### ISSUE TITLE
Odyssey Stats: global variable name collision on WoA

### ISSUE THREAD

**Issue by brbrr** [Automattic/jetpack#28141]
### Impacted plugin

Jetpack

### Steps to Reproduce

1. Make sure Odyssey Stats is enabled in Jetpack Settings
2. Visit /wp-admin/admin.php?page=stats on your test WoA site
3. Check the JS console, and see the error

### A clear and concise description of what you expected to happen.

No errors I guess 🦐 

### What actually happened

```
help-center.min.js?ver=ff2c78958a39f67ef213:43
Uncaught TypeError: Cannot redefine property: state
at Function.defineProperty ()
```

### Platform (Simple, Atomic, or both?)

Atomic

### Reproducibility

Consistent

### Severity

None

### Available workarounds?

None

**Comment by jsnmoon** [comment]
This should be addressed via #28156; we should be shepherding Atomic customers to Calypso Stats instead of Jetpack/Odyssey Stats. Nonetheless, it'd be good to prevent this from occurring.

**Comment by jsnmoon** [comment]
I believe this should be resolved via https://github.com/Automattic/wp-calypso/pull/71283.
